isisd/isis_adjacency.c

```diff
@@ -207,6 +207,22 @@ static const char *adj_state2string(int state)
 	return NULL; /* not reached */
 }
 
+static const char *adj_level2string(int level)
+{
+	switch (level) {
+	case 1:
```

Comment:
case IS_LEVEL1, IS_LEVEL2 and IS_LEVEL_1_AND_2 please

Reply:
will do